src/video_core/buffer_cache/buffer_cache.h

```diff
@@ -482,6 +606,10 @@ class BufferCache {
     u64 modified_ticks = 0;
 
     std::vector<u8> staging_buffer;
+    std::list<MapInterval> marked_for_unregister;
+
+    std::shared_ptr<std::unordered_set<MapInterval>> uncommited_flushes{};
```

Comment:
```suggestion
    std::shared_ptr<std::unordered_set<MapInterval>> uncommitted_flushes;
```

std::shared_ptr already has a constructor. `{}` is unnecessary.